cmd/kwild/server/build.go

```diff
@@ -412,7 +412,7 @@ func buildDB(d *coreDependencies, closer *closeFuncs) *pg.DB {
 	// If yes, restore the database from the snapshot
 	restoreDB(d)
 
-	db, err := d.dbOpener(d.ctx, d.cfg.AppCfg.DBName, 11)
+	db, err := d.dbOpener(d.ctx, d.cfg.AppCfg.DBName, 24)
```

Comment:
Eh, I was actually just fooling with this, but I think this his a righteous and safe change for the main DB to have a larger reader pool.  The default `max_connections` setting in every stock postgresql.conf I have encountered for 8 years has been 100.   Further `kwild` code requires this `postgres` setting to be at least 50:

https://github.com/kwilteam/kwil-db/blob/8176ce17d408ddbf611ee8595e7593b9bd7b146b/internal/sql/pg/system.go#L116

So we could possibly go even higher, but let's be cautious.